package/setup.py

```diff
@@ -0,0 +1,25 @@
+import subprocess
+
+import setuptools
+
+poetry_result = subprocess.run(
+    ["poetry", "version"],
+    check=True,
+    stdout=subprocess.PIPE,
+    encoding="utf-8",
+)
+poetry_output = poetry_result.stdout.strip()
+
+name, version = poetry_output.split(" ")
```

Comment:
Default `.split` separator is whitespace. Did you explicitly want just spaces?

Reply:
We can probably get away with any whitespace here.